scripts/zkai_attention_block_statement_bridge_gate.py

```
@@ -0,0 +1,717 @@
+    if accumulator.get("all_mutations_rejected") is not True:
+        raise AttentionBlockStatementBridgeError("d128 accumulator mutations are not all rejected")
+    if "make attention output feed the block receipt input under one statement commitment" not in _list(
+        surface.get("next_required_work"), "one-block next_required_work"
+    ):
+        raise AttentionBlockStatementBridgeError("one-block surface no longer names the bridge follow-up")
+
+    bridge_contract = _dict(attention.get("bridge_contract"), "attention bridge_contract")
+    metrics = _dict(bridge_contract.get("metrics"), "attention metrics")
+    comparisons = _dict(metrics.get("comparisons"), "attention comparisons")
+    if any(value is not True for value in comparisons.values()):
+        raise AttentionBlockStatementBridgeError("attention bridge comparisons are not all true")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)